integrations/elasticsearch/metadata.yaml

```diff
@@ -3,12 +3,9 @@ short_name: Elasticsearch
 display_name: Elasticsearch
 description: 'Elasticsearch is an open-source search server, based on the
 
-  [Lucene](https://lucene.apache.org/) search library. It runs in a Java virtual
+  [Lucene](https://lucene.apache.org/){: .external} search library. It runs in a Java
```

Comment:
@sophieyfang what are these `{: .external}` annotations? This doesn't look like standard markdown.

Reply:
These are the ones required by Mary Tabasko for the google public page.